src/Command/CacheWarmupCommand.php

```diff
@@ -53,6 +53,7 @@ protected function configure(): void
             ->addOption('schemas', 's', InputOption::VALUE_OPTIONAL, 'Skip caching schemas during cache warmup', false)
             ->addOption('endpoints', 'p', InputOption::VALUE_OPTIONAL, 'Skip caching endpoints during cache warmup', false)
             ->addOption('removeOnly', 'r', InputOption::VALUE_OPTIONAL, 'Only do the remove data from cache during this command', false)
+            ->addOption('cacheOnly', 'b', InputOption::VALUE_OPTIONAL, 'Only cache objects from a specific bundle', false)
```

Comment:
cacheOnly might not be the correct name, I would expect something in this name referencing the fact that we can only cache for one bundle.